### ISSUE TITLE
inadequate documentation for set()

### ISSUE THREAD

**Issue by timrobinson33** [boblauer/MockDate#40]
I'm not sure whether I'm using it wrong or whether it just plain doesn't work.

Say I want to set the current time to be 1pm on 15th June in Chicago
```
MockDate.set("2020-06-15T13:00:00", -5 * 60)
console.log(new Date().toString())
==>  Mon Jun 15 2020 13:00:00 GMT+0100 (British Summer Time)
```
It seems the value I'm passing into set is assumed to be specified relative to my current time zone (London in summertime), which kind of defeats the point of mocking, because I want my tests to run the same whichever time zone they run in.

I think the most people would assume the date you specify is be interpreted relative to the timezone you specify.

**Edit: there's something else funny going on as well:**

I worked around the previous problem by specifying the time zone when parsing the date like this:
```
MockDate.set("2020-06-15T13:00:00-05:00", -5 * 60)
const date = new Date()
console.log(date.toUTCString())
==> Mon, 15 Jun 2020 18:00:00 GMT
```
So far so good - 1pm in chicago is 6pm GMT - but now look at this:
```
console.log(`local hours ${date.getHours()} UTC hours ${date.getUTCHours()}
                offset=${date.getTimezoneOffset() / 60} hours`)
==> local hours 19 UTC hours 18 offset=-5 hours
```
The local hours is wrong, even though both the UTC hours and the offset are correct

**Comment by boblauer** [comment]
The 2nd parameter `timezoneOffset` does not behave like you would expect, and it's something I plan on removing in the future, as it causes a lot of confusion such as this and was a mistake to add at all.

The only thing it does is change the result of `getTimezoneOffset`, it doesn't actually modify the date. There is no way for this package to behave as if the computer was running in a completely different timezone, nor do I intend to support that use case, as it's a huge undertaking and not the focus of what this library is trying to achieve.

Sorry, not the answer you were looking for I'm sure.

**Comment by timrobinson33** [comment]
OK thanks, I just wanted to make sure it wasn't me being dumb.

It seems there isn't a bullet-proof mock for date/time/timezone out there which surprises me a bit but it's obviously more difficult than I thought.

FWIW I ended up using `timezone-mock` to mock just the timezone, but that wouldn't play nice with your MockDate so I decided to just inject a date constructor function into my app, which is probably a good practice anyway.
